streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopicManager.java

```diff
@@ -206,10 +206,6 @@ public void makeReady(final Map<String, InternalTopicConfig> topics) {
                     cause instanceof LeaderNotAvailableException) {
                     // This topic didn't exist or leader is not known yet, proceed to try to create it
                     log.debug("Topic {} is unknown, hence not existed yet.", topicName);
-                } else {
-                    log.error("Unexpected error during topic description for {}.\n" +
-                        "Error message was: {}", topicName, cause.toString());
-                    throw new StreamsException(String.format("Could not create topic %s.", topicName), cause);
```

Comment:
Should we still log it, perhaps as a warning?

If I understand the background, this case is unexpected except with 0.10 brokers, so it seems like swallowing it could mask an important condition.